html/apps/tweetfeed/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  background-color: gray !important;
+}
+
+md-content.md-default-theme, md-content {
+    color: rgba(0,0,0,0.87);
+    background-color: rgb(66, 66, 66);
+}
+
+img.md-user-avatar{
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Element (img.md-user-avatar) is overqualified, just use .md-user-avatar without element name.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2597868539/issues/source?bid=3288751&fileBranchId=3307409#l10)